### PR TITLE
Added option to remove all breakpoints

### PR DESCRIPTION
**Pull request by sagorika1996**
Associated Issue: #3575 

### Summary of Changes

* Added option to remove currently selected breakpoint
* Added option to remove all breakpoints

ezgif com-video-to-gif 2

### CODE REVIEW

File: assets/panel/debugger.properties [hunk 1]
```diff
@@ -221,6 +221,14 @@ breakpointMenuItem.enableAll=Enable all breakpoints
 breakpointMenuItem.disableAll=Disable all breakpoints
 breakpointMenuItem.deleteAll=Remove all breakpoints
 
+# LOCALIZATION NOTE (breakpointMenuItem.deleteSelf): Access key to remove the 
+# currently selected breakpoint from the context menu
+breakpointMenuItem.deleteAll.accesskey=r
```
Comment (jasonLaster) [inline assets/panel/debugger.properties]: should this be  `deleteSelf`

File: assets/panel/debugger.properties [hunk 2]
```diff
@@ -221,6 +221,14 @@ breakpointMenuItem.enableAll=Enable all breakpoints
 breakpointMenuItem.disableAll=Disable all breakpoints
 breakpointMenuItem.deleteAll=Remove all breakpoints
 
+# LOCALIZATION NOTE (breakpointMenuItem.deleteSelf): Access key to remove the
```
Comment (jasonLaster) [inline assets/panel/debugger.properties]: `breakpointMenuItem.deleteSelf.accesskey`

**Comment by jasonLaster** [comment]
@sagorika1996 small l10n things, but otherwise looks great

File: src/components/SecondaryPanes/Breakpoints.js [hunk 3]
```diff
@@ -73,6 +74,44 @@ class Breakpoints extends PureComponent {
...
+
+    e.preventDefault();
+
+    const removeBreakpointLabel = L10N.getStr("breakpointMenuItem.deleteSelf");
+    const removeAllBreakpointsLabel = L10N.getStr(
+      "breakpointMenuItem.deleteAll"
+    );
+
+    const removeBreakpointKey = L10N.getStr(
+      "breakpointMenuItem.deleteAll.accesskey"
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Breakpoints.js]: Should this be deleteSelf?

Comment (sagorika1996) [inline src/components/SecondaryPanes/Breakpoints.js]: Done.

**Comment by jasonLaster** [comment]
Thanks so much!

**Pull request merged**